java/src/main/java/com/jongallant/azure/identity/extensions/AzureIdentityCredentialAdapter.java

```diff
@@ -0,0 +1,77 @@
+package com.jongallant.azure.identity.extensions;
```

Comment:
Can you check the physical path on this?  Looks like jongallant is still in the directory structure.

Reply:
Yes, this might have been because of the 2 parallel PRs. I have fixed this now.